docs/qunit-options.md

```diff
@@ -54,8 +54,20 @@ Default: `false`
 
 Fail a test when the global namespace is polluted. See the [QUnit cookbook](http://qunitjs.com/cookbook/#discussion-id170) for more information.
 
-# Filtering specs
+# Command line options
 
-**Filtering by module name**:
+## Filtering by module name: `--modules`
 
-`grunt qunit --modules=foo` will run the module `foo`. You can specify one or multiple, comma-separated modules to run.
+`grunt qunit --modules="foo"`
+
+Will run the module `foo`. You can specify one or multiple, comma-separated modules to run.
+
+## Running tests in seeded-random order: `--seed`
+
+`grunt qunit --seed="a-string"`
+
+Tells `QUnit` to run tests in a seeded-random order. The value provided will be used as the seed in a pseudo-random number generator to ensure that results are reproducible. The randomization will also respect the reorder option if enabled and re-run failed tests first without randomizing them.
```

Comment:
This seems like it should "just" link to the right QUnit docs instead. At least shorten these paragraphs?